Missing authorization field in Kamelet model

Leads to errors when creating and verifying Kamelet sources

```
org.citrusframework.yaks.camelk.KameletSteps.loadKameletFromFile(java.lang.String) test-slack-source-buuahkta2aa1v2m3317g test       com.consol.citrus.exceptions.TestCaseFailedException: An error has occurred.
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "authorization" (class org.citrusframework.yaks.camelk.model.KameletSpec), not marked as ignorable (5 known properties: "sources", "dependencies", "definition", "flow", "types"])
```

--- a/java/steps/yaks-camel-k/src/main/java/org/citrusframework/yaks/camelk/model/KameletSpec.java
+++ b/java/steps/yaks-camel-k/src/main/java/org/citrusframework/yaks/camelk/model/KameletSpec.java
@@ -32,7 +32,7 @@ import com.fasterxml.jackson.module.jsonSchema.types.ObjectSchema;
 import io.fabric8.kubernetes.api.model.KubernetesResource;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"definition", "dependencies", "types", "sources", "flow"})
+@JsonPropertyOrder({"definition", "dependencies", "types", "sources", "authorization", "flow"})
 @JsonDeserialize(
         using = JsonDeserializer.None.class
 )
@@ -46,6 +46,8 @@ public class KameletSpec implements KubernetesResource {
     private Map<String, TypeSpec> types;
     @JsonProperty("sources")
     private List<Source> sources;
+    @JsonProperty("authorization")
+    private AuthorizationSpec authorization;
     @JsonProperty("flow")
     private Map<String, Object> flow;
 
@@ -63,6 +65,14 @@ public class KameletSpec implements KubernetesResource {
 
     public void setSources(List<Source> sources) {
         this.sources = sources;
+    }
+
+    public AuthorizationSpec getAuthorization() {
+        return authorization;
+    }
+
+    public void setAuthorization(AuthorizationSpec authorization) {
+        this.authorization = authorization;
     }
 
     public List<String> getDependencies() {
@@ -270,5 +280,9 @@ public class KameletSpec implements KubernetesResource {
                 this.example = example;
             }
         }
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class AuthorizationSpec {
     }
 }

